library/src/scala/compiletime/package.scala

```diff
@@ -26,12 +26,8 @@ package object compiletime {
    *  ```
    *  or
    *  ```scala
-   *  error(code"My error of this code: ${println("foo")}")
-   *  ```
-   *  or
-   *  ```scala
    *  inline def errorOnThisCode(inline x: Any) =
-   *    error(code"My error of this code: $x")
+   *    error("My error of this code: " + codeOf(x))
```

Comment:
If the s-interpolator won't work, that should be mentioned in the docs (but also maybe we should create an issue about this since it seems like a good thing to support)

Reply:
It would be good to support it. I will create an issue.

Reply:
#10315